autoload/ale/fixers/biome.vim

```diff
@@ -1,17 +1,10 @@
-" Author: Akiomi Kamakura <akiomik@gmail.com>
-" Description: Fixing files with biome (ex.rome).
-
 function! ale#fixers#biome#Fix(buffer) abort
     let l:executable = ale#handlers#biome#GetExecutable(a:buffer)
-    let l:options = ale#Var(a:buffer, 'javascript_biome_options')
-    let l:node = ale#Var(a:buffer, 'javascript_biome_node_executable')
+    let l:options = ale#Var(a:buffer, 'biome_options')
 
     return {
-    \   'command': (has('win32') ? (ale#Escape(l:node) . ' ') : '')
-    \       . ale#Escape(l:executable)
-    \       . ' check --apply'
-    \       . ale#Pad(l:options)
-    \       . ' %t',
-    \   'read_temporary_file': 1,
+    \   'command': '%e format'
```

Comment:
Sorry for commenting on an already merged PR, but to me it seems like `%e` is not substituted with the executable when I run this. Switching to use `l:executable` directly works fine. Is `%e` supposed to work in this context?

Reply:
@lanker I created a PR for this issue. #4733